validator/controller.go

```diff
@@ -107,6 +102,21 @@ func (c *controller) ListenToEth1Events(cn pubsub.SubjectChannel) {
 	}
 }
 
+// initShares initializes shares
+func (c *controller) initShares(options ControllerOptions) {
+	if options.CleanRegistryData {
```

Comment:
@amir-blox maybe move to the cli package?

Reply:
Same comment as above:

The delete is not "generic" - it uses some prefix, meaning we need to perform this from the actual collection/storage where the prefix is defined